tests/runners/airflow/tasks/test_spark_task.py

```diff
@@ -15,17 +15,108 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+
+import os
+import tempfile
 from unittest import TestCase
 
+from liminal.build import liminal_apps_builder
+from liminal.kubernetes import volume_util
 from liminal.runners.airflow import DummyDag
+from liminal.runners.airflow.executors.kubernetes import KubernetesPodExecutor
 from liminal.runners.airflow.tasks.spark import SparkTask
+from tests.util import dag_test_utils
 
 
 class TestSparkTask(TestCase):
     """
     Test Spark Task
     """
 
+    _VOLUME_NAME = 'myvol1'
+
+    def test_spark_on_k8s(self):
```

Comment:
Let's add a similar test to test the spark image builder. Without airflow, but with docker